gameplay/src/Matrix.cpp

```diff
@@ -906,12 +906,12 @@ void Matrix::transformPoint(const Vector3& point, Vector3* dst) const
 void Matrix::transformVector(Vector3* vector) const
 {
     GP_ASSERT(vector);
-    transformVector(vector->x, vector->y, vector->z, 0.0f, vector);
+    transformVector(vector->x, vector->y, vector->z, 1.0f, vector);
```

Comment:
Oh, I see now. This is to rotate a vector ignoring translation. But it also scale the vector...
This is kind of a confusing behavior and function name.